third-party/realdds/src/dds-device.cpp

```diff
@@ -153,4 +153,15 @@ std::shared_ptr< extrinsics > dds_device::get_extrinsics( std::string from, std:
     return empty;
 }
 
+bool dds_device::supports_metadata() const
+{
+    for( const auto & option : _impl->_options )
+    {
+        if( option->get_name() == "metadata_enabled" )
```

Comment:
Not `_md_supported`?

Reply:
It is not only supported by the device, but enabled and the server can send it to us.
Will change to `metadata-enabled`

Reply:
Added to the to-do list a task to change from device level options to device_info flag. And to consider using per-stream flags instead of device level

Reply:
`device_info` as in the device-info topic?
I think metadata needs to be per-stream and never at the device-level.

Reply:
Implementer per-stream